### PR TITLE
Some unit tests.

### PR DESCRIPTION
**Pull request by amshuman-kr**
A first simple unit test using ginkgo+gomega.

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -9,204 +9,150 @@
...
 [[projects]]
   name = "github.com/Azure/azure-sdk-for-go"
-  packages = [
-    "arm/compute",
-    "arm/disk",
-    "arm/network"
-  ]
-  revision = "eae258195456be76b2ec9ad2ee2ab63cdda365d9"
-  version = "v12.2.0-beta"
+  packages = ["arm/compute","arm/disk","arm/network"]
```
[the remark this answers is not in the document]
Comment (amshuman-kr) [reply Gopkg.lock]: Fixed after upgrading to dep 0.4 and revendoring.

File: Gopkg.lock [hunk 2]
```diff
@@ -227,27 +174,50 @@
...
 
 [[projects]]
   name = "github.com/matttproud/golang_protobuf_extensions"
   packages = ["pbutil"]
-  revision = "fc2b8d3a73c4867e51861bbdd5ae3c1f0869dd6a"
+  revision = "3247c84500bff8d9fb6d579d800f20b3e091582c"
+  version = "v1.0.0"
+
+[[projects]]
+  name = "github.com/modern-go/concurrent"
```
[the remark this answers is not in the document]
Comment (amshuman-kr) [reply Gopkg.lock]: I have added them in LICENSE.md. Concurrent and reflect2 do not seem to have any copyright statements.

**Comment by amshuman-kr** [comment]
@msohn I have made changes as you suggested. Can you PTAL?

**Comment by prashanth26** [comment]
@msohn - Can you please approve this PR so that I can merge it.

**Comment by amshuman-kr** [comment]
@msohn Your review is still marked as blocked.

I have fixed the license mention for new packages as well as fixed the formatting of Gopkg.lock using the latest version of dep.

Is there anything else you wanted me to work on?

**Comment by amshuman-kr** [comment]
@msohn Thanks for the review and approval!

**Pull request merged**